pkg/helpers/helpers.go

```diff
@@ -160,3 +161,21 @@ func ConcatSlices[T any](slices ...[]T) []T {
 	}
 	return tmp
 }
+
+// TODO(SNOW-1058419): address during identifiers rework
```

Comment:
I think this is not the right issue to be linked here.

Reply:
Fixed.